commands/bootstrap/setup.go

```diff
@@ -75,7 +75,8 @@ func setupYorcServer(workingDirectoryPath string) error {
 		WorkersNumber:    inputValues.Yorc.WorkersNumber,
 		Infrastructures:  inputValues.Infrastructures,
 		Terraform: config.Terraform{
-			PluginsDir: workDirAbsolutePath,
+			PluginsDir:         workDirAbsolutePath,
+			KeepGeneratedFiles: true,
```

Comment:
Why?

Reply:
To ease the bootstrap failure troubleshooting, did set ansible.KeepGeneratedRecipes too